lib/services/services_lsb.h

```diff
@@ -13,7 +13,7 @@
 G_GNUC_INTERNAL int services__get_lsb_metadata(const char *type, char **output);
 G_GNUC_INTERNAL GList *services__list_lsb_agents(void);
 G_GNUC_INTERNAL bool services__lsb_agent_exists(const char *agent);
-G_GNUC_INTERNAL int services__lsb_prepare(svc_action_t *op);
+G_GNUC_INTERNAL int services__lsb_prepare(const svc_action_t *op);
```

Comment:
ditto, not const in the header either

Can revert copyright if it hasn't been modified this year otherwise

Also should these all be in `services_private.h`?

Reply:
It's fine to have multiple library-private headers. In this case separating standard-specific APIs is logically clean and makes it easy to enable conditional compilation in the makefiles (even though we don't currently do that for LSB).